src/applications/edu-benefits/tests/complaint-tool/config/schoolInfo.unit.spec.js

```diff
@@ -11,14 +11,17 @@ import formConfig from '../../../complaint-tool/config/form';
 
 describe('complaint tool school info', () => {
   const { schema, uiSchema } = formConfig.chapters.schoolInformation.pages.schoolInformation;
+  delete uiSchema.school.facilityCode;
```

Comment:
Now that I'm not hiding the search field, I need to be able to render the page without setting up a redux store to test `formConfig`. Simplest approach was to just remove the part of config that's not being tested. 🤷‍♂️